style.css

```diff
@@ -356,8 +365,66 @@ th{
   cursor: pointer;
 }
 
+/*styling the contact form*/
+.input_box{
```

Comment:
Styling for the admin controls in settings page. This helps save time as I won't need to style each form individually.